tests/Integration/ConcurrencyTest.php

```diff
@@ -53,8 +53,10 @@ protected static function createEventStore(): EventStore
     private static function connection(): Connection
     {
         if (self::$connection === null) {
+            require __DIR__ . '/../../vendor/symfony/polyfill-php85/bootstrap.php';
             require __DIR__ . '/../../vendor/laravel/framework/src/Illuminate/Support/helpers.php';
             require __DIR__ . '/../../vendor/laravel/framework/src/Illuminate/Collections/helpers.php';
+            require __DIR__ . '/../../vendor/laravel/framework/src/Illuminate/Collections/functions.php';
```

Comment:
Those changes are unrelated, but I needed these lines to make `composer test:concurrency` work